tx/textual/valuerenderer/valuerenderer.go

```diff
@@ -111,28 +115,31 @@ func (r *Textual) GetMessageValueRenderer(md protoreflect.MessageDescriptor) (Va
 	fullName := md.FullName()
 	vr, found := r.messages[fullName]
 	if found {
-		return vr, nil
+		var fd protoreflect.FieldDescriptor
+		return vr(fd), nil
```

Comment:
@amaurym is this fine this way?

Reply:
Hmm, no it seems incorrect to me.

Let's wait for #13604 to get merged first, and then we can revisit this.